corehq/motech/repeaters/dbaccessors.py

```diff
@@ -153,22 +154,33 @@ def _get_repeater_ids_by_domain(domain):
     return [result['id'] for result in results]
 
 
-def iterate_repeat_records(due_before, chunk_size=10000, database=None):
+def iterate_repeat_records_for_ids(doc_ids):
     from .models import RepeatRecord
-    json_now = json_format_datetime(due_before)
+    return (RepeatRecord.wrap(doc) for doc in get_docs(RepeatRecord.get_db(), doc_ids))
```

Comment:
could be worth using [iter_docs](https://github.com/dimagi/commcare-hq/blob/799d28f253a3dacd16c1b99ff2e526591532fefe/corehq/ex-submodules/dimagi/utils/couch/database.py#L61) instead of `get_docs`